.circleci/test.sh

```diff
@@ -20,6 +20,7 @@ echo "Running Python Tests"
 
 echo "Running MNIST Test"
 python test/test_train_mnist.py --tidy
+python test/test_train_mp_mnist_amp.py
```

Comment:
maybe
```
if [ -x "$(command -v nvidia-smi)" ]; then
  python test/test_train_mp_mnist_amp.py
fi
```
we don't need to run this test for cpu

Reply:
Done. BTW, it could run on CPU.